en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,183 @@
+ 
+- Updated occurrence: result indicator period target @value attribute (`#355 <https://github.com/IATI/IATI-Schemas/issues/355>`__) (`discussion link <https://discuss.iatistandard.org/t/results-represent-more-than-quantitative-data-included-2-03/872>`__)
+
+- Updated occurrence: result indicator period actual element (`#356 <https://github.com/IATI/IATI-Schemas/issues/356>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
+
+- Updated occurrence: result indicator period actual @value attribute (`#358 <https://github.com/IATI/IATI-Schemas/issues/358>`__) (`discussion link <https://discuss.iatistandard.org/t/results-represent-more-than-quantitative-data-included-2-03/872>`__)
+
+- Added new document-link element (as a child of various result elements)(`#359 <https://github.com/IATI/IATI-Schemas/issues/359>`__) (`discussion link <https://discuss.iatistandard.org/t/add-document-link-to-results-indicator-included-2-03/895>`__)
+ 
+- Added new reference element (as a child of the result element) (`#362 <https://github.com/IATI/IATI-Schemas/issues/362>`__) (`discussion link <https://discuss.iatistandard.org/t/results-vocabulary-attribute-option-included-2-03/879>`__)
+ 
+- Added new @aggregation-status attribute for the indicator element (`#363 <https://github.com/IATI/IATI-Schemas/issues/363>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new location element (as a child of the baseline element) (`#365 <https://github.com/IATI/IATI-Schemas/issues/365>`__) (`discussion link <https://discuss.iatistandard.org/t/results-improve-consistency-of-results-standard-included-2-03/874>`__)
+ 
+- Added new dimension element (as a child of the baseline element) (`#366 <https://github.com/IATI/IATI-Schemas/issues/366>`__) (`discussion link <https://discuss.iatistandard.org/t/results-allow-disaggregations-of-results-data-included-2-03/871>`__)
+  
+Organisations Schema
+-------------------
+
+- Updated definition: organisation-identifier element (`#361 <https://github.com/IATI/IATI-Schemas/issues/361>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
+
+Activities and Organisations Schemas
+-------------------
+  
+- Updated definition: all @xml:lang attributes (`#336 <https://github.com/IATI/IATI-Schemas/issues/336>`__) (`discussion link <https://discuss.iatistandard.org/t/language-recommend-use-of-iso-639-1-included-2-03/842>`__)
+
+- Updated definition: [reporting/participating/receiver/provider]-org @ref attribute (`#339 <https://github.com/IATI/IATI-Schemas/issues/339>`__) (`discussion link <https://discuss.iatistandard.org/t/migration-of-organisationregistrationagency-codelist-to-org-id-guide-included-2-03/851>`__)
```

Comment:
Specifying the `-org` part like this is unclear. Suggest stating the whole element name each time.

Reply:
I don't know if I'm just used to looking at it this way, but it seems clear to me, though the slashes maybe need a rethink to avoid looking like regular XPaths.

Reply:
309fd857672ab6c51bf7c09d10d8ae60f0c06a3c